core-services/prow/02_config/_config.yaml

```diff
@@ -194,6 +194,7 @@ tide:
     - openshift/kubernetes
     - openshift/kube-rbac-proxy
     - openshift/kube-state-metrics
+    - openshift/kube-storage-version-migrator
```

Comment:
Inclusion in this stanza only applies for `release-foo` branches -- do you want to also do `master`? Add to the other list in here, then, too

Reply:
I've updated to also include in a section that excludes the `release-fooo` branches.